zt_backend/router.py

```diff
@@ -588,7 +594,135 @@ def delete_item(delete_request: request.DeleteItemRequest):
             status_code=500, detail=f"An unexpected error occurred: {str(e)}"
         )
 
+def initialize_mime_types():
+    """Initialize and customize MIME types."""
+    mimetypes.init()
+
+    custom_mime_types: Dict[str, str] = {
+        '.md': 'text/markdown',
+        '.pptx': 'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+        '.xlsx': 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+        '.docx': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        '.7z': 'application/x-7z-compressed',
+        '.rar': 'application/vnd.rar',
+        '.webp': 'image/webp',
+        # Add more custom MIME types here as needed
+    }
+
+    for ext, mime_type in custom_mime_types.items():
+        mimetypes.add_type(mime_type, ext)
+
+def get_mime_type(filename: str) -> str:
```

Comment:
Lets break this all out into a utils file. In the same vein any helper functions for file uploads/changes should be moved there. We want to keep the router as only endpoints as much as possible

Reply:
okay , I will update this